tests/pass-dep/libc/libc-mem.rs

```diff
@@ -241,6 +241,42 @@ fn test_reallocarray() {
     }
 }
 
+#[cfg(not(any(target_os = "windows", target_os = "wasi")))]
```

Comment:
Why not on wasi?

Reply:
Where have you seen that? libc even declares aligned_alloc for wasi so I'd think that it exists there.